.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Type of pull request
+
+* [ ] Bug fix (involves code and configuration changes)
+* [ ] New feature (involves code and configuration changes)
+* [ ] Documentation update
+* [ ] Something else
+
+# About
+
+*Please describe with a few words what this pull request is about*
```

Comment:
Maybe this should be a comment in the rendered page to avoid people to let it like that instead of removing it
```suggestion
<!-- Please describe with a few words what this pull request is about -->
```